src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountExtractPdpEntryServiceImpl.java

```diff
@@ -0,0 +1,139 @@
+            fullName = removeLastCharacterWhenComma(fullName);
+        }
+        return fullName;
+    }
+    
+    private String removeLastCharacterWhenComma(String fullName) {
+        if (fullName.lastIndexOf(KFSConstants.COMMA) >= getPayeeNameFieldSize()-2) {
+            fullName = fullName.substring(0, fullName.lastIndexOf(KFSConstants.COMMA));
+        }
+        return fullName;
+    }
+
+    @Override
+    public PdpFeedPayeeIdEntry buildPayeeIdEntry(ConcurStandardAccountingExtractDetailLine line) {
+        PdpFeedPayeeIdEntry payeeIdEntry = new PdpFeedPayeeIdEntry();
+        payeeIdEntry.setContent(line.getEmployeeId());
+        if (StringUtils.equalsIgnoreCase(line.getEmployeeStatus(), ConcurConstants.StandardAccountingExtractPdpConstants.EMPLOYEE_STATUS_CODE)) {
+            payeeIdEntry.setIdType(ConcurConstants.StandardAccountingExtractPdpConstants.EMPLOYEE_PAYEE_STATUS_TYPE_CODE);
+        } else {
+            payeeIdEntry.setIdType(ConcurConstants.StandardAccountingExtractPdpConstants.NON_EMPLOYEE_PAYEE_STATUS_TYPE_CODE);
+        }
+        return payeeIdEntry;
+    }
+
+    @Override
+    public PdpFeedDetailEntry buildPdpFeedDetailEntry(ConcurStandardAccountingExtractDetailLine line) {
+        PdpFeedDetailEntry currentDetailEntry  = new PdpFeedDetailEntry();
+        currentDetailEntry.setSourceDocNbr(buildSourceDocumentNumber(line.getReportId()));
+        currentDetailEntry.setFsOriginCd(ConcurConstants.StandardAccountingExtractPdpConstants.FS_ORIGIN_CODE);
+        currentDetailEntry.setFdocTypCd(ConcurConstants.StandardAccountingExtractPdpConstants.DOCUMENT_TYPE);
+        currentDetailEntry.setInvoiceNbr("");
+        currentDetailEntry.setPoNbr("");
+        currentDetailEntry.setInvoiceDate(formatDate(line.getBatchDate()));
+        return currentDetailEntry;
+    }
+    
+    @Override
+    public String buildSourceDocumentNumber(String reportId) {
+        String sourceDocNumber = ConcurConstants.StandardAccountingExtractPdpConstants.DOCUMENT_TYPE + reportId;
+        return StringUtils.substring(sourceDocNumber, 0, ConcurConstants.SOURCE_DOCUMENT_NUMBER_FIELD_SIZE);
+    }
+
+    @Override
+    public PdpFeedAccountingEntry buildPdpFeedAccountingEntry(ConcurStandardAccountingExtractDetailLine line) {
+        PdpFeedAccountingEntry currentAccountingEntry =  new PdpFeedAccountingEntry();
+        currentAccountingEntry.setCoaCd(line.getChartOfAccountsCode());
+        currentAccountingEntry.setAccountNbr(line.getAccountNumber());
+        currentAccountingEntry.setObjectCd(line.getJournalAccountCode());
+        currentAccountingEntry.setSubObjectCd(line.getSubObjectCode());
+        currentAccountingEntry.setOrgRefId(line.getOrgRefId());
+        currentAccountingEntry.setProjectCd(line.getProjectCode());
+        currentAccountingEntry.setAmount(KualiDecimal.ZERO.toString());
+        return currentAccountingEntry;
+    }
+    
+    protected String formatDate(Date date) {
```

Comment:
Ah yes, I forgot that the DateTimeService has a public method that accepts a date pattern. That would work for this, so that we're not relying on the Rice default pattern.